src/cli/src/cluster/mod.rs

```diff
@@ -1,63 +1,54 @@
+use structopt::StructOpt;
+
 mod install;
 mod uninstall;
 mod util;
 mod check;
 mod releases;
 
-pub use process::process_cluster;
-
-use structopt::StructOpt;
-
-pub use install::InstallCommand;
-use uninstall::UninstallCommand;
-use check::CheckCommand;
-use releases::ReleasesCommand;
+use install::InstallOpt;
+use uninstall::UninstallOpt;
+use check::CheckOpt;
+use releases::ReleasesCmd;
+use crate::Result;
 
-#[allow(clippy::large_enum_variant)]
 #[derive(Debug, StructOpt)]
 #[structopt(about = "Available Commands")]
-pub enum ClusterCommands {
+pub enum ClusterCmd {
     /// Install a Fluvio cluster, locally or on Minikube
     #[structopt(name = "install")]
-    Install(InstallCommand),
+    Install(Box<InstallOpt>),
```

Comment:
Why Box?

Reply:
Clippy gives a warning when one enum variant is much larger than the others. It appears `InstallCommand` is much larger. Clippy recommends boxing it, and structopt seems to work with it just fine.